CSharp/examples/BermudanSwaption.csproj

```diff
@@ -182,9 +130,8 @@
   </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PreBuildEvent>copy "$(SolutionDir)cpp\bin\$(Platform)\$(Configuration)\NQuantLibc.dll" "$(TargetDir)"
-copy "$(SolutionDir)csharp\bin\$(Platform)\$(Configuration)\NQuantLib.dll" "$(TargetDir)"</PreBuildEvent>
+    <PreBuildEvent>copy "$(SolutionDir)cpp\bin\$(Platform)\$(Configuration)\NQuantLibc.dll" "$(TargetDir)"</PreBuildEvent>

```

Comment:
Do the examples run without copying the second DLL?

Reply:
Yes, the "trick" is the project reference between `c#` projects within the `*.csproj` files. It takes care of copying the project output across.

Unfortunately I don't think this work for mixed `c#` and `c++` projects as well. At least I didn't manage to get this up and working in our projects by now. That's why we have to copying the output across as a build action.

```xml
  <ItemGroup>
    <ProjectReference Include="..\csharp\NQuantLib.csproj">
      <Project>{928f98ee-7d50-457f-9304-a6818dcf1079}</Project>
      <Name>NQuantLib</Name>
    </ProjectReference>
  </ItemGroup>
```

Reply:
Ok, thanks.

Reply:
I never tried to find a proper solution for this as we used the build event for it, but I have to admit there is a way to do this: [How do I add a reference to an unmanaged C++ project called by a C# project?](https://stackoverflow.com/a/42507529)

And it works! But as you @lballabio have merged already, I keep it as a secret ;-)

I cannot test it anyhow on an older version then VS 2019...